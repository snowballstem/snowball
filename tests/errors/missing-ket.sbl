externals (stem
integers (i
define stem as (
  non-()
  $(cursor > 0
  $i = (1 + 2
  hop 5 [tolimit] delete
