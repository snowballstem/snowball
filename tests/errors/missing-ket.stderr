errors/missing-ket.sbl:2: ) omitted after name
errors/missing-ket.sbl:3: ) omitted after name
errors/missing-ket.sbl:4: name omitted after -
errors/missing-ket.sbl:6: ) omitted after number
errors/missing-ket.sbl:7: ) omitted after number
errors/missing-ket.sbl:8: ) omitted after delete
